homeassistant/components/spotify/browse_media.py

```diff
@@ -363,7 +363,7 @@ async def build_item_response(  # noqa: C901
                     items.append(_get_track_item_payload(playlist_item.track))
                 elif playlist_item.track.type is ItemType.EPISODE:
                     if TYPE_CHECKING:
-                        assert isinstance(playlist_item.track, SimplifiedEpisode)
+                        assert isinstance(playlist_item.track, Episode)
```

Comment:
Not completely sure why `SimplifiedEpisode` doesn't cause the errors with the Python version set to 3.12. However, using the concrete class is in line with the case above for `Track`
https://github.com/home-assistant/core/blob/18de735619f696da714668067dc5e331f39711a5/homeassistant/components/spotify/browse_media.py#L360-L363